### ISSUE TITLE
A problem completing Making Headlines quest 

### ISSUE THREAD

**Issue by Magnum1965** [EdenServer/community#1729]
### Checklist

- [ ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [ ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)

**Comment by Magnum1965** [comment]
After getting to the Mahogany door, I recieved a message saying CoP was not linked to my account, and could not continue and finish the quest. Any ideas? All CS were completed.

**Comment by Slamm21** [comment]
did you use the eden installer? and have you done any COP mission yet?

**Comment by Magnum1965** [comment]
Yes I used the Eden installation and I did the first cop mission

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Xerlic1212** [comment]
I also cannot complete this quest. 

I have all 4 scoops. I went back to Naiko-Paneiko and he wants me to confirm the Cat Burglar's hideous is under Horutoto Ruins. I go to the Mahogany door and I get the error about CoP not installed. Clicking the door again gives the message "The door is firmly closed."

**Comment by Althaia** [comment]
Please note that this is a text-only issue! You can still complete the quest upon returning to the chief editor of the newspaper as intended.

I have encountered the exact same thing as others have noted here, where it has an error message about CoP and clicking the door again gives the message about the door being firmly closed.

I'm in a Windurstian linkshell, so I know many people who have tried this too, and it is apparently a known issue -- known to be text-only -- among Windurstians.

**Comment by ShiyoKozuki** [comment]
Still bugged and I cannot complete it

**Comment by Althaia** [comment]
I talked to Shiyo and apparently she and her friend both have all four required key items, checked the door and got the "CoP not installed" message, and returned to Naiko-Paneiko, just like I had -- as well as other Windurstians whom I've talked to in the past -- and are not able to complete the quest. Even though I (and others) could complete it in the past.

My thought is that "Scooped!" was added on the Dec. 30th patch and that adding that may've screwed something up for "Making Headlines," because they're both at the same NPC.

Good luck!

**Comment by aprilgarrity** [comment]
I can confirm; I go to the Mahogany Door and receive the message: "You are unable to proceed to the next area because Chains of Promathia is not registered to your account." After inspecting the door again, I repeatedly receive the message "The door is firmly closed." I then returned to Windurst Waters in attempt to turn in the quest anyway, as Althaia advised, however, I cannot turn in the quest and it still does not register as being completed.

**Comment by Althaia** [comment]
Gosh, I wonder why there's such a problem with this now.

The best explanation, in my mind, is that when the coder added "Scooped!" for the same NPC, it may've screwed something up for "Making Headlines."

Alternatively, it's possible I didn't click it again after the CoP message, since I already knew from other people saying it in chat that the CoP message was the "correct" message.

If that's the case, then perhaps clicking the door more than once somehow clears it from your account, so that you can't complete the quest?

**Comment by Balrahn** [comment]
I'm getting the same problem saying I don't have CoP installed. Returning to the quest giver does not finish the quest.

**Comment by jaymantan** [comment]
I'm having the exact same issue.
